src/classes/TDTM_Global_API.cls

```diff
@@ -235,7 +243,7 @@ global class TDTM_Global_API {
             this.targetObject = targetObject;
             this.actions = actions;
             this.active = active;
-            this.loadOrder = loadOrder;
```

Comment:
Same here. I think we also need to populate the old value for external code querying getting a value. (For example, if the real value is 1.2, they'll get 1. But better than getting null.)